### DISCUSSION TITLE
Remix Cloudflare Pages

### DISCUSSION THREAD

**Discussion by alexanderbartels** [mantinedev/mantine#1220] [Q&A] [answered]
Hi, thanks for this library. I wanted to give mantine a try in a new project. I followed the remix setup guide on your website: https://mantine.dev/getting-started/ and added the dark/light theme example from the remix example i found here: https://github.com/remix-run/remix/tree/main/examples/mantine

But it looks like that server side rendering is not working together with cloudflare pages. I get the following error if i run `npm run dev` or `npm run build` locally:

```
> start
> cross-env NODE_ENV=production npm run dev:wrangler


> dev:wrangler
> cross-env NODE_ENV=development wrangler pages dev ./public

🚧 'wrangler pages <command>' is a beta command. Please report any issues to https://github.com/cloudflare/wrangler2/issues/new/choose
Compiling worker to "/var/folders/j7/q...bdq04c0000gp/T/functionsWorker.js"...
Compiled Worker successfully.
[pages:err] ReferenceError: Buffer is not defined
    at /Users/xyz/functions/[[path]].js:17:2222
    ....
    at SourceTextModule.evaluate (node:internal/vm/module:224:23)
Could not start Miniflare.

If you think this is a bug then please create an issue at https://github.com/cloudflare/wrangler2/issues/new.
```

Is there an easy way to use mantine ssr with remix and cloudflare pages? 

For now i removed `injectStylesIntoStaticMarkup(markup)` from `entry.server.tsx` so i can continue developing. 

Thanks
Alex

**Comment by rtivital** [comment, marked as answer]
We do not officially support Cloudflare workers, but there is an example with esbuild + Remix – https://github.com/aiji42/remix-esbuild-override/tree/main/examples/mantine-cloudflare

**Reply by alexanderbartels** [reply]
thanks for the fast response. Works perfectly fine!

**Reply by asharnadeem** [reply]
This link leads to a 404 page. Is there an updated example for 2024?

**Reply by rtivital** [reply]
v7 should just work fine without any setup.
